tests/test_user.py

```diff
@@ -82,6 +82,11 @@ def test_platform(index, name, factory):
 
     assert user.platform == name
 
+def test_can_write_private_message(factory):
```

Comment:
```suggestion
@pytest.mark.parametrize('value, expected', [
    (0, False),
    (1, True),
])
def test_can_write_private_message(factory, value, expected):
```

желательно также в тестах добавлять значение которое может принимать поле `can_write_private_message`